packages/turbo-workspaces/src/managers/npm.ts

```diff
@@ -0,0 +1,188 @@
+    paths: {
+      root: args.workspaceRoot,
+      packageJson: path.join(args.workspaceRoot, "package.json"),
+      lockfile: path.join(args.workspaceRoot, "package-lock.json"),
+      nodeModules: path.join(args.workspaceRoot, "node_modules"),
+    },
+    workspaceData: {
+      globs: packageJson.workspaces || [],
+      workspaces: expandWorkspaces({
+        workspaceGlobs: packageJson.workspaces,
+        ...args,
+      }),
+    },
+  };
+}
+
+/*
+ Create npm workspaces from generic format
+ Creating npm workspaces involves:
+
+  1. Adding the workspaces field in package.json
+  2. Setting the packageManager field in package.json
+  3. Updating all workspace package.json dependencies to ensure correct format
+*/
+async function create(args: CreateArgs): Promise<void> {
+  const { project, options, to, logger } = args;
+
+  logger.mainStep(`Creating npm workspaces`);
+  const packageJson = getPackageJson({ workspaceRoot: project.paths.root });
+  logger.rootHeader();
+  if (project.packageManager !== "yarn") {
+    logger.rootStep(
+      `adding "workspaces" field to ${project.name} root "package.json"`
+    );
+    packageJson.workspaces = project.workspaceData.globs;
+  }
+  logger.rootStep(
+    `adding "packageManager" field to ${project.name} root "package.json"`
+  );
+  packageJson.packageManager = `${to.name}@${to.version}`;
+
+  if (!options?.dry) {
+    fs.writeJSONSync(project.paths.packageJson, packageJson, { spaces: 2 });
+  }
+
+  // if we're converting from pnpm, we need to update the workspace package.json files
+  if (project.packageManager === "pnpm") {
+    // root dependencies
+    updateDependencies({
+      workspace: { name: "root", paths: project.paths },
+      project,
+      to,
+      logger,
+      options,
+    });
+
+    logger.workspaceHeader();
+    project.workspaceData.workspaces.forEach((workspace) =>
+      updateDependencies({ workspace, project, to, logger, options })
+    );
+  }
+}
+
+/*
+Remove npm workspace data
+
+Removing npm workspaces involves:
+  1. Removing the workspaces field from package.json
+*/
+async function remove(args: RemoveArgs): Promise<void> {
+  const { project, options, to, logger } = args;
+
+  logger.mainStep(`Removing npm workspaces`);
+  if (to.name !== "yarn") {
+    const packageJson = getPackageJson({ workspaceRoot: project.paths.root });
+    delete packageJson.workspaces;
+    logger.subStep(
+      `removing "workspaces" field in ${project.name} root "package.json"`
+    );
+
+    if (!options?.dry) {
+      fs.writeJSONSync(project.paths.packageJson, packageJson, { spaces: 2 });
+
+      // collect all workspace node_modules directories
+      const allModulesDirs = [
+        project.paths.nodeModules,
+        ...project.workspaceData.workspaces.map((w) => w.paths.nodeModules),
+      ];
+      try {
+        logger.subStep(`removing "node_modules"`);
+        await Promise.all(
+          allModulesDirs.map((dir) => fs.rm(dir, { recursive: true }))
```

Comment:
What about `force: true` to avoid the `ENOENT` check?

```suggestion
          allModulesDirs.map((dir) => fs.rm(dir, { recursive: true, force: true }))
```